Runtime/Native/iOS/NativeClient.cs

```diff
@@ -77,7 +80,7 @@ public NativeClient(string gameObjectName, BacktraceConfiguration configuration)
         /// Start crashpad process to handle native Android crashes
         /// </summary>
 
-        private void HandleNativeCrashes(BacktraceConfiguration configuration)
+        private void HandleNativeCrashes(BacktraceConfiguration configuration, List<string> attachments)

```

Comment:
Why aren't the attachments part of the configuration object?

Reply:
answer above.